schemas/kbc.schema.json

```diff
@@ -27,25 +31,29 @@
       "format":"date-time"
     },
     "credentialSchema":{
-      "type":"object",
-      "properties":{
-        "id":{
-          "type":"string",
-          "format":"uri"
+      "type":"array",
```

Comment:
I don't anticipate multiple, but web5-go produces an array even when there's 1 schema. https://github.com/TBD54566975/web5-go/blob/main/vc/vc.go#L117-L127

Reply:
FWIW web5-rs does not treat this as an array. Proposal: we type it as an array but everywhere in our current use cases only assumes one item, therefore we don't have to concern ourselves with the processing rules, and thereafter we can add whatever processing rules we come up with, without introducing a breaking change.

I guess taking a step back, @decentralgabe what do you think about making a prescription on our spec here, to not be an array, and instead and object?

Seeing this in both 1.1 and 2.0

> The value of the credentialSchema [property](https://www.w3.org/TR/vc-data-model-2.0/#dfn-property) MUST be one or more data schemas

And then 2.0 has this particular language

> If multiple schemas are present, validity is determined according to the processing rules outlined by each associated credentialSchema type property.

But unclear to me why we would ever want more than one JSON schema?

Reply:
a credential could contain multiple pieces of data ...

like 
1. a name -- associated with a schema for a first, middle, last name
2. employment data -- associated with a schema for start date, title, place of employment

in which case there could be multiple non-overlapping terms, which is a valid use case.

however, we do not have any processing logic in our SDKs to handle
1. conflicting terms between the schema (imagine different regexes for the same property names in two schemas)
2. how to handle validation when some schemas pass and others do not

until we do 1 and 2 (and maybe more) I believe we're better off restricting the data models to 1 schema

Reply:
alright cool, so for now, I propose we stick with an array, I'll add a task to do so in rust, but our processing of the VC assumed an array of size 1

@tomdaffurn on a slight tangential, I noticed web5-go's [VC verify function](https://github.com/TBD54566975/web5-go/blob/c22ede228fb5ec5172bb79ffa840d18b0dce249d/vc/vcjwt.go#L98) doesn't do anything with the schema right now

Reply:
Thanks @KendallWeihe https://github.com/TBD54566975/web5-spec/issues/161

Reply:
Assuming 1 schema for current use cases works for me. It's sort of necessary because of incompatibility between Prsentation Defnition's Field array-contains syntax and JSONPath

Reply:
Happy to merge @decentralgabe ?

Reply:
I would recommend adding `maxItems` to the array property to enforce the constraint
https://json-schema.org/understanding-json-schema/reference/array#length

Reply:
done @decentralgabe